src/usr/share/gpgmailer/gpgmailmessage.py

```diff
@@ -94,6 +90,18 @@ def set_body(self, body):
         self._check_if_saved()
         self.message['body'] = body
 
+    # Adds a list of recipients.
```

Comment:
This should probably say something about e-mail addresses. Actually, the method name should probably say something about e-mail addresses.